src/react/components/pages/export/exportPage.tsx

```diff
@@ -74,6 +74,7 @@ export default class ExportPage extends React.Component<IExportPageProps> {
         };
 
         await this.props.actions.saveProject(projectToUpdate);
+        await this.props.actions.exportProject(projectToUpdate);
```

Comment:
FYI - We don't want this here right now.  It is also causing your CI test failure.  You can run export from the editor page by clicking the "Export" button in the toolbar.